input/fsh/omrs-patient.fsh

```diff
@@ -3,33 +3,59 @@ Parent:       Patient
 Id:           omrs-patient
 Title:        "OpenMRS Patient"
 Description:  "A FHIR Patient as understood by OpenMRS"
-* identifier 1..*
+* identifier 0..*
 * identifier ^slicing.discriminator.type = #value
 * identifier ^slicing.discriminator.path = "system"
 * identifier ^slicing.rules = #closed
 * identifier only OMRSPatientIdentifier
-* active 1..1
+* active 1..1 MS
 * active = true
-* name 1..1
-* gender 1..1
-* birthDate 1..1
-* maritalStatus 0..0
-* multipleBirth[x] 0..0
+* name 1..* MS
```

Comment:
@sherrif10 any reason you had to change those values in this Profile ??

Reply:
Sorry , No reason at all, i reverted them back. thanks.